src/api-reference/financial-integration/v4.financial-integration-service-use-cases.markdown

```diff
@@ -666,3 +666,41 @@ Credit Card Vendor|-|USD 284
 [**Standard Accounting Extract**](./v4.financial-integration-use-cases-extract-information.html#eccccepno)
 
 [**JSON Example**](json/FI-API-ReportID-7F00502973D84DEFA49E-Example-6.2.4.json)
+
+### The use case examples below illustrate Funds and Grants Integration (FGI) scenarios and the related records of Funds Reservation and Funds Check. The use case documentation is not meant to be a comprehensive list of all use cases and FGI scenarios. However, the JSON from the Financial Integration Service will provide all the required data to support all potential financial use cases.
+
```

Comment:
@njkmr0027 You have this styled as a heading, but it's really just a paragraph. Does it need to be bold? Could we style it as a note? Let me know and I can fix the formatting.